apps/demo-ui/run-ui.sh

```diff
@@ -59,9 +72,11 @@ cd "${UIDIR}"
 # Running the UI this way means that the html is unminified and hence easy to read
 # Since the UI is open source, there's little downside of doing this and it helps with
 # debugging.
+# See this link for why we're using RSA self-signed certificates for npm:
+# https://community.letsencrypt.org/t/ecdsa-certificates-not-supported-by-create-react-app/201387/9
 BROWSER=none PORT=3001 WDS_SOCKET_PORT=3000 HTTPS=true \
-SSL_CRT_FILE="${PROXYDIR}/${HOST}-cert.pem" \
-SSL_KEY_FILE="${PROXYDIR}/${HOST}-key.pem" \
+SSL_CRT_FILE="${PROXYDIR}/${SELFSIGNED}-cert.pem" \
```

Comment:
If we have a real key, why are we using selfsigned here?

Reply:
There's a comment on L75.  For some reason, npm doesn't support ECDSA.